qu19041_ex3.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import matplotlib.pyplot as pyplot
+
+# PART (A)
+
+# Here we first define the function to be integrated followed by the function to evaluate the 
+# integral for conditions c.
+def integrand(x_screen, x_aperture, c):
+    return np.exp( c['waveno'] * 1j * (x_screen - x_aperture)**2 / (2*c['screendist']) )
+
+def fresnel_integral(x_screen, c):
+
+    assert c['intervals'] % 2 == 0, "number of intervals must be even."
+
+    x_aperture = np.linspace(c['xapt_lower'], c['xapt_upper'], c['intervals']+1)
+    dx_aperture = x_aperture[1] - x_aperture[0]
```

Comment:
Like here, for instance. If you had x_aperture[1]-x_aperture[0] it would look bad. So, I prefer to have a space around any operator * / + - **